### PR TITLE
Add color_extract filter

### PR DESCRIPTION
**Pull request by hishamco**

### CODE REVIEW

File: Fluid/Filters/ColorFilters.cs [hunk 1]
```diff
@@ -79,6 +79,44 @@ public static FluidValue ToHsl(FluidValue input, FilterArguments arguments, Temp
...
+            }
+            else if (RgbColor.TryParse(value, out rgbColor))
+            {
+                hslColor = HslColor.FromRgb(rgbColor);
+            }
+            else if (HslColor.TryParse(value, out hslColor))
+            {
+                rgbColor = RgbColor.FromHsl(hslColor);
+                // TODO: Check why 1 is added to RGB
+                rgbColor = new RgbColor(rgbColor.R - 1, rgbColor.G - 1, rgbColor.B - 1, rgbColor.A);
```
Comment (hishamco) [inline Fluid/Filters/ColorFilters.cs:100]: Don't merge unless I fix a fractions issue that increases RGB by 1

Comment (sebastienros) [reply Fluid/Filters/ColorFilters.cs:100]: Convert.ToInt32?

Comment (hishamco) [reply Fluid/Filters/ColorFilters.cs:100]: I already did, but I will try to check what's going on HSL to RGB algorithm

**Comment by hishamco** [comment]
It's ready to merge Seb

**Pull request merged**